dynamics-nav-app/LocalFunctionality/Germany/how-to-set-up-delivery-reminders.md

```diff
@@ -15,7 +15,7 @@
     ms.author: edupont
 
 ---
-# How to: Set Up Delivery Reminders
+# How to: Set Up Delivery Reminders 
```

Comment:
Change to "How to Set Up Deliver Reminders (Germany)" to be consistent with added title.

Reply:
Done.